### PR TITLE
Update to 2.4.52 for Aarch64

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Looking through the changelogs, it seems that version 2.4.50 added support for new "architectures", which I presume is why aarch is passing now.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+set -x
+export CPPFLAGS="${CPP_FLAGS} -I$PREFIX/include "
```
Comment (isuruf) [inline recipe/build.sh:2]:
```suggestion
export CPPFLAGS="${CPPFLAGS} -isystem $PREFIX/include "
```

Comment (hmaarrfk) [reply recipe/build.sh]: I wonder if this is actually needed now.

**Comment by hmaarrfk** [comment]
Thanks you two!

**Pull request merged**